doc/contributing/hw/racl/README.md

```diff
@@ -1,64 +1,92 @@
-If the access is denied then the error is logged into the Error-Log register.
-Source processing elements may have a static role assignment or can support a dynamic role assignment, like in this example.
-
-### RACL rules
-1. The IP may group the set of registers in the IP based on this access control model into one or more control groups.
-   For each control group there will be a RACL policy picked from one of the [standard RACL policies](#racl-policies).
-   An IP should not use a RACL policy other than those specified here.
-
-2. Each RACL takes the form of a pair of permission bitmaps.
-   For each RACL implemented by the IP there must be two permission bitmap registers - ReadPerm and WritePerm where each bit in the register corresponds to a role.
-   These two registers must be mapped to the control network (CTN) outside of the RoT and must be only readable and writable by the RoT role, i.e., be part of a special ROT_PRIVATE policy (see [RACL policies](#racl-policies)).
-   These registers should have reset default defined at design time but these can be updated by RoT at reset if required.
-
-|                 | Role 0 | Role 1 | Role 2 | Role 3 | Role 4 | Role 5 | Role 6 | Role 7 |
-|-----------------|:------:|:------:|:------:|:------:|:------:|:------:|:------:|:------:|
-| *Bit position:* | *0*    | *1*    | *2*    | *3*    | *4*    | *5*    | *6*    | *7*    |
-| `WritePerm`     |        |        |        |        |        |        |        |        |
-| `ReadPerm`      |        |        |        |        |        |        |        |        |
-
-3. Each ReadPerm bit indicates if the corresponding role (see column header) has read permission to the group of registers controlled by this ACL.
-   Each WritePerm bit indicates if the corresponding role has write permission.
-   When the bit is 1, the corresponding permission is provided and when it is 0 it is denied.
+At the target, the role and the attributes of the transaction (Read/Write, address, and size) are used to determine if the access is allowed or denied.
+If the access is denied then the error is logged into an Error-Log register.
+
+### RACL Policies
+Each policy register takes the form of a pair of permission bitmaps, `ReadPerm` and `WritePerm`, where each bit corresponds to a role.
+E.g., if 16 different roles are defined, the register looks as follows, where the bit position of each role is defined by the role encoding that is specified via the `role_id` in the [configuration file](#top-level-racl-configuration).
+
+|                 | Role 0 | Role 1 | ... | Role 14 | Role 15 |
+|-----------------|:------:|:------:|:---:|:-------:|:-------:|
+| *Bit position:* | *0*    | *1*    | ... | *14*    | *15*    |
+| `WritePerm`     |        |        |     |         |         |
+| `ReadPerm`      |        |        |     |         |         |
```

Comment:
I know that this isn't new, but I wonder whether it makes sense to give example values for the two permission rows here. At the moment, this is just a list of headings!

I'd also suggest adding "..." on the other rows as well or picking few enough roles that no "..." is needed.